Update rs-7-2-4-52.md (#2807)
RedisLabs/redislabs-docs: content/rs/release-notes/rs-7-2-4-releases/rs-7-2-4-52.md

#### Access control deprecations

> - The following predefined roles and Redis ACLs are no longer available with new Redis Enterprise Software version 7.2.4 clusters. if they are not associated with any users or databases in the cluster:

Reviewer: We can probably remove the second part if we're only talking about new clusters.
```suggestion
- The following predefined roles and Redis ACLs are no longer available for new Redis Enterprise Software version 7.2.4 clusters:
```

    - Custom roles (not management roles): Cluster Member, Cluster Viewer, DB Member, DB Viewer, None.

    - Redis ACLs: Not Dangerous and Read Only.
      

> - In the upcoming maintenance releases, the roles and ACLs will be automatically removed during the upgrade to Redis Enterprise Software version 7.2.4, unless they are associated with any users or databases in the cluster.

Reviewer: ```suggestion
- In upcoming maintenance releases, the deprecated roles and ACLs will be removed automatically when you upgrade to Redis Enterprise Software version 7.2.4, unless they are associated with any users or databases in the cluster.
```
